context/InfoContext.js

```diff
@@ -10,75 +11,75 @@ export function InfoProvider({children}) {
   const [tripEvents, setTripEvents] = useState(null);//trip events for one trip
   const [tripid, setTripid] = useState(null);//trip id
 
-  const { userData, userToken } = useContext(AuthContext)
+  const { userData, userToken } = useContext(AuthContext);
+
+  const authHeader = {
+    'Accept': 'application/json, text/plain, */*', 
+    'Content-Type': 'application/json',
+    'Authorization': `Bearer ${userToken}`
+  }
+
+  // should I import this from Auth context?
```

Comment:
could create a utils folder for stuff like this I suppose

Reply:
yeah maybe